apps/auth-server/config/index.js

```diff
@@ -61,3 +61,19 @@ exports.client = {
   defaultEmail: '',
   defaultProjectUrl: ''
 }
+
+// use the same setup as the API
+exports.datetime = {
```

Comment:
Maybe add a comment here that this config is used in `Intl.DateTimeFormat` for later reference?